.github/workflows/ci.yml

```
@@ -131,13 +134,13 @@ jobs:
         env:
           CHECK_FORMATTING: '${{ matrix.check_formatting }}'
           NO_TEST_REQUIREMENTS: '${{ matrix.no_test_requirements }}'
-      - if: always()
+      - if: ${{ always() && matrix.check_formatting != '1' }}
```

How about dropping the mustache?

```suggestion
      - if: >-
          always()
          & & matrix.check_formatting != '1'
```